<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(10, 38, 71);</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>651</width>
     <height>392</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 107, 135);
color: rgb(0, 0, 0);</string>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rating</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NoOfRatingsandReviews</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DiscountedPrice</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Delivery</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Percentageoff</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="number_txt">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>560</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>201</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 38, 71);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(24, 61, 61, 255), stop:0.471591 rgba(24, 61, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 255, 255);
font: 16pt &quot;Mongolian Baiti&quot;;
border-radius: 5px;


</string>
   </property>
   <property name="text">
    <string>Flight Number</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fType_txt">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>630</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Mongolian Baiti&quot;;
border-radius: 5px;


</string>
   </property>
   <item>
    <property name="text">
     <string>Domestic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Commercial</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>620</y>
     <width>201</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 38, 71);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(24, 61, 61, 255), stop:0.471591 rgba(24, 61, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 255, 255);
font: 16pt &quot;Mongolian Baiti&quot;;
border-radius: 5px;


</string>
   </property>
   <property name="text">
    <string>Flight Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>680</y>
     <width>201</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 38, 71);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(24, 61, 61, 255), stop:0.471591 rgba(24, 61, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 255, 255);
font: 16pt &quot;Mongolian Baiti&quot;;
border-radius: 5px;


</string>
   </property>
   <property name="text">
    <string>Piolt Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pName">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>680</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>740</y>
     <width>201</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 38, 71);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(24, 61, 61, 255), stop:0.471591 rgba(24, 61, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 255, 255);
font: 16pt &quot;Mongolian Baiti&quot;;
border-radius: 5px;


</string>
   </property>
   <property name="text">
    <string>Capacity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="capacity_txt">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>740</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>800</y>
     <width>201</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 38, 71);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(24, 61, 61, 255), stop:0.471591 rgba(24, 61, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 255, 255);
font: 16pt &quot;Mongolian Baiti&quot;;
border-radius: 5px;


</string>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ss">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>790</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dd">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>560</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>570</y>
     <width>201</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 38, 71);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(24, 61, 61, 255), stop:0.471591 rgba(24, 61, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 255, 255);
font: 16pt &quot;Mongolian Baiti&quot;;
border-radius: 5px;


</string>
   </property>
   <property name="text">
    <string>Destinaton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>640</y>
     <width>201</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 38, 71);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(24, 61, 61, 255), stop:0.471591 rgba(24, 61, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 255, 255);
font: 16pt &quot;Mongolian Baiti&quot;;
border-radius: 5px;


</string>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_txt">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>630</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>710</y>
     <width>201</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 38, 71);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(24, 61, 61, 255), stop:0.471591 rgba(24, 61, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 255, 255);
font: 16pt &quot;Mongolian Baiti&quot;;
border-radius: 5px;


</string>
   </property>
   <property name="text">
    <string>Duaration</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="duaration_txt">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>710</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>680</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBt">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>770</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 38, 71);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(24, 61, 61, 255), stop:0.471591 rgba(24, 61, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 255, 255);
font: 16pt &quot;Mongolian Baiti&quot;;
border-radius: 5px;


</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_bt">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>810</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 38, 71);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(24, 61, 61, 255), stop:0.471591 rgba(24, 61, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 255, 255);
font: 16pt &quot;Mongolian Baiti&quot;;
border-radius: 5px;


</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
